src/pages/Login-Signup/login.jsx

```diff
@@ -0,0 +1,72 @@
+import "../../root.css";
+import "../Login-Signup/auth.css";
+import { useAuth } from "../../context/authContext.js";
+import { Link, useNavigate } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+const Login = () => {
+const { token, loginHandler } = useAuth();
+const navigate = useNavigate();
+const [ userDetails, setUserDetails] = useState({email: "", password: ""});
+
+// const guestUserDetails = (e) =>{
+//     e.preventDefault();
+// setUserDetails({
+// email: "adarshbalika@gmail.com",
+// password: "adarshBalika123"
+// })
+// }
+
+const guestUserHandler = (event) => {
+    event.preventDefault();
+    setUserDetails({
+        email: "adarshbalika@gmail.com",
+        password: "adarshBalika123",
+    })
+}
+
+function formHandler(e){
+    e.preventDefault();
+      loginHandler(userDetails);
+}
+
+// useEffect(()=>{
+// if(token){
+// navigate("/homepage");
+// }
+// },[token,navigate]);
```

Comment:
Similarly can remove this block too if not required. I guess you are trying to implement this feature right related to authenticated users trying to access login and signup routes then they shall be navigated somewhere else right?

Reply:
Yes, and then forget to remove it.